### ISSUE TITLE
Suggestion: Friends Page

### ISSUE THREAD

**Issue by Cortlandd** [haskell-lang/haskell-lang#102]
Similar to the [Friends of Rust Page](https://www.rust-lang.org/en-US/friends.html) that showcases companies using Rust and their use case. I think this would be pretty cool. Not sure how I should approach implementing it though. I don't want to just statically type it all. I think that would be bad for page loading speed. Any suggestions?

**Comment by bitemyapp** [comment]
Haskell.org links to these pages:

http://industry.haskell.org/partners

https://github.com/commercialhaskell/commercialhaskell#companies

**Comment by snoyberg** [comment]
Sounds like a good idea to me. Are you interested in taking a first stab at a PR for this?

**Comment by Cortlandd** [comment]
@snoyberg yeah I am. But as I said, I'm not sure how I should approach? To statically implement them all would result in slower page loading correct? And other than statically implementing company logos, etc. I'm not sure how to approach.

**Comment by snoyberg** [comment]
I'm not sure what you mean by static typing in this case, but I doubt it
will affect load time. If you want to start with a small version of the
page, we can provide feedback of there's anything inefficient.

**Comment by bitemyapp** [comment]
I don't think @Cortlandd means static _typing_ — I think he means a static web page versus one that incrementally loads the content or images with JS.

Thing is, if the concern is logos — browsers already handle lazily loading images in parallel separately of text content. I don't think any JavaScript is needed here. Apologies if I misread.

**Comment by Cortlandd** [comment]
@bitemyapp You are correct. So simply typing it all in will work? And won't affect load time.

**Comment by bitemyapp** [comment]
@Cortlandd yeah just type it all in, mimic an existing page that does something like what you want.
